queue/queue.go

```diff
@@ -76,10 +77,10 @@ func (r *RabbitMQ) Consume(messageChannel chan amqp.Delivery) {
 
 	go func() {
 		for message := range incomingMessage {
-			log.Println("Incoming new message")
+			internal.Logger.Info("Incoming new message")
 			messageChannel <- message
 		}
-		log.Println("RabbitMQ channel closed")
+		internal.Logger.Info("RabbitMQ channel closed")
```

Comment:
it's better to put the message after closing or conjugate the verb correctly, because it wasn't closed at that time